### PR TITLE
Test/stripe user api

### PR DESCRIPTION
**Pull request by xXValhallaCoderXx**
- Basic Stripe API Calls
- Basic User Info Calls
- Flask CORS added

### CODE REVIEW

File: Server/instance/settings.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+SQLALCHEMY_DATABASE_URI ='postgresql://scc:xXValhallaXx@postgres:5432/scc'
+
+# Seed User DB Details
+SEED_ADMIN_EMAIL = 'odin@admin.com'
+SEED_ADMIN_PASSWORD = '123'
+ADMIN = True
+
+# Stripe API
+STRIPE_PUBLISHABLE_KEY = 'pk_test_TkjSo0wKO7TMxZ0lCQHwqqHf '
```
Comment (kolyaak) [inline Server/instance/settings.py]: Please, don't upload keys to github.

And this is Horangi keys.
Horangi keys must not be shared outside of the company at all.

**Comment by xXValhallaCoderXx** [comment]
Sorry,
This settings.py was meant to be in gitignore,

Also just to confirm,
That means our thesis mates can't access the test data either?

On 4 Jun 2017 10:59, "kolyaak" <notifications@github.com> wrote:

*@kolyaak* commented on this pull request.

Guys, please be careful with keys
------------------------------

In Server/instance/settings.py
<https://github.com/horangi-ir/customer_success_portal/pull/4#discussion_r119999460>
:

> @@ -0,0 +1,12 @@
+DEBUG = True
+
+SQLALCHEMY_DATABASE_URI ='postgresql://scc:xXValhallaXx@postgres:5432/scc'
+
+# Seed User DB Details
+SEED_ADMIN_EMAIL = 'odin@admin.com'
+SEED_ADMIN_PASSWORD = '123'
+ADMIN = True
+
+# Stripe API
+STRIPE_PUBLISHABLE_KEY = 'pk_test_TkjSo0wKO7TMxZ0lCQHwqqHf '

Please, don't upload keys to github.

And this is Horangi keys.
Horangi keys must not be shared outside of the company at all.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/horangi-ir/customer_success_portal/pull/4#pullrequestreview-41931964>,
or mute the thread

**Comment by kolyaak** [comment]
Yes. There is less risk in accessing test data but better to stay safe. How about creating a new stripe test account? 

Please, remove this keys asap.

**Comment by xXValhallaCoderXx** [comment]
How do you mean another Stripe test account? It won't be linked to Horangi?
What data can we use?

What will be easier?
Do we not let group mates have any access to server side code (even though
it's independent from web backend already)?

What is the main issue you're concerned with? Them seeing the Stripe key?
Or them seeing Stripe test data??

Will remove the settings.py straight away either way and wait your reply

**Comment by kolyaak** [comment]
Let's take this discussion to another place. You know, this is the public repository.

**Pull request merged**